### PR TITLE
[NETBEANS-54] Module Review

### LINKED ISSUE #119: Cleanup NetBeans license header for module jshell.support
Review and cleanup the license header in jshell.support

Oracle donated NetBeans to the Apache Software Foundation.

Your task now is to *replace* the old license header that has Oracle (and/or Sun Microsystems
and/or NetBeans) in it with the standard Apache Software Foundation license header.

For an empty file that supports headers you must *add* the Apache license header.

Note that you have to use the proper license header for Java, .properties or HTML files.
Look at the nearby files for templates and copy-paste the license header.

When done commit the changes with a commit message that starts with first line
"[NETBEANS-54] Module Review <module>" and a message body describing your review.

See a longer explanation about what we are doing here: https://cwiki.apache.org/confluence/display/NETBEANS/List+of+Modules+to+Review

Please announce in a comment when you have started work.

Files that need to be looked at in jshell.support:
```
jshell.support/src/org/netbeans/modules/jshell/project/Bundle.properties
jshell.support/src/org/netbeans/modules/jshell/resources/ShellClass.java.template
jshell.support/src/org/netbeans/modules/jshell/resources/consoleExample.jsh
jshell.support/src/org/netbeans/modules/jshell/tool/Feedback.java
jshell.support/test/unit/src/org/netbeans/modules/jshell/support/reploutput.txt
jshell.support/test/unit/src/org/netbeans/modules/jshell/support/reploutput_golden.txt

```

### PR DESCRIPTION
**Pull request by michael-simons**
Fixes #119 

I did not change

- jshell.support/test/unit/src/org/netbeans/modules/jshell/support/reploutput.txt
- jshell.support/test/unit/src/org/netbeans/modules/jshell/support/reploutput_golden.txt

as I think that this will break tests.

I did not change

- jshell.support/src/org/netbeans/modules/jshell/resources/consoleExample.jsh

as I don't know how that template is used and what format a comment must have.

### CODE REVIEW

File: jshell.support/src/org/netbeans/modules/jshell/resources/ShellClass.java.template [hunk 1]
```diff
@@ -1,3 +1,21 @@
+<#--
```
Comment (emilianbold) [inline jshell.support/src/org/netbeans/modules/jshell/resources/ShellClass.java.template]: I believe the user will see this with Tools | Templates, then Open in Editor. So, perhaps it should *not* have a license header?

**Review by emilianbold — CHANGES_REQUESTED** [review]

**Comment by michael-simons** [comment]
That might be true, I’m not sure. The license would not be rendered when executed though because it’s a free marker template. I can change it.

**Comment by emilianbold** [comment]
I've asked on the dev@ mailing list about this.

I know it won't generate anything when executed. Still, as an user I would be bothered by that huge license header.

**Comment by emilianbold** [comment]
If you want to commit already, exclude the .template file from the PR and we can leave that for later, after we talk about it on dev@

**Comment by michael-simons** [comment]
I've deleted the header. I would find it annoying and confusing, too. Sorry for the hiccup.

**Review by emilianbold — APPROVED** [review]

**Pull request merged**